src/test/java/com/plaid/client/integration/AssetReportAuditCopyRemoveTest.java

```diff
@@ -0,0 +1,48 @@
+package com.plaid.client.integration;
+
+import com.plaid.client.PlaidClient;
+import com.plaid.client.request.AssetReportAuditCopyCreateRequest;
+import com.plaid.client.request.AssetReportAuditCopyRemoveRequest;
+import com.plaid.client.request.common.Product;
+import com.plaid.client.response.AssetReportAuditCopyCreateResponse;
+import com.plaid.client.response.AssetReportAuditCopyRemoveResponse;
+import com.plaid.client.response.AssetReportCreateResponse;
+import org.junit.Test;
+import retrofit2.Response;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class AssetReportAuditCopyRemoveTest extends AbstractItemIntegrationTest {
+
+  @Override
+  protected List<Product> setupItemProducts() {
+    return Arrays.asList(Product.ASSETS);
+  }
+
+  @Override
+  protected String setupItemInstitutionId() {
+    return TARTAN_BANK_INSTITUTION_ID;
+  }
+
+  @Test
+  public void testAssetReportAuditCopyCreteSuccess() throws Exception {
```

Comment:
did you want Remove here

Reply:
can you please go through your test names and make sure they correspond what they are testing